IETF-RFC.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
   <?xml-stylesheet type="text/xsl" href="rfc2629.xslt" ?>
-  <!-- generated by https://github.com/cabo/kramdown-rfc version 1.7.29 (Ruby 3.2.3) -->
+  <!-- generated by https://github.com/cabo/kramdown-rfc version 1.7.29 (Ruby 3.3.8) -->
```

Comment:
Mahdi just pushed a CI workflow to generate the xml file, you may want to drop this file.